### DISCUSSION TITLE
https://kurianbenoy.com/posts/2021/2021-03-21-qualities_software_developers.html

### DISCUSSION THREAD

**Discussion by giscus** [kurianbenoy/kurianbenoy.github.io#28] [Q&A]
# https://kurianbenoy.com/posts/2021/2021-03-21-qualities_software_developers.html

I am pretty new to the field of software engineering and my journey as a developer has just begun.

https://kurianbenoy.com/posts/2021/2021-03-21-qualities_software_developers.html

**Comment by kurianbenoy** [comment]
Just trying out this for testing
